external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSourceSuite.scala

```diff
@@ -318,6 +318,84 @@ class KafkaSourceSuite extends KafkaSourceTest {
     )
   }
 
+  test("union bug in failover") {
```

Comment:
nit: `test("SPARK-22956: currentPartitionOffsets should be set when no new data comes in")`

Reply:
DONE